src/vaults/vaults.utils.ts

```diff
@@ -378,10 +385,11 @@ export async function loadVaultEventPerformances(
     try {
       const emittedVault = getVaultDefinition(chain, tokenEmitted.address);
       const vaultValueSources = await getVaultCachedValueSources(emittedVault);
-      // these value sources are already saved in apy formats for vault compounding
-      const compoundingSource = vaultValueSources.find((source) => source.type === SourceType.Compound);
+      // search for the persisted apr variant of the compounding vault source, if any
+      const compoundingSource = vaultValueSources.find((source) => source.type === SourceType.PreCompound);
       if (compoundingSource) {
-        const compoundingSourceApy = (1 + (emissionApr * compoundSource.apr) / periods) ** periods - 1;
+        const compoundingSourceApy =
+          ((1 + (emissionApr * compoundingSource.apr) / (periods * 100)) ** periods - 1) * 100;
```

Comment:
both these sources are stored human readable so they need an extra divison (and mult after) by 100 in the calculations